third_party/upb/WORKSPACE

```diff
@@ -1,6 +1,7 @@
 workspace(name = "upb")
```

Comment:
This one may also need to be removed?

Reply:
Like above, I expect this will be ignored.